APP/app.test.js

```diff
@@ -109,3 +109,31 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /app/articles/:article_id/comments", () => {
+  it("Gets all comments for an article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toEqual(11);
+        body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
+        });
+      });
+  });
+
+  it("Responds with a 404 error when ID doesn't exist", () => {
+    return request(app)
+      .get("/api/articles/3453/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
+      });
```

Comment:
Great sad path here

You're missing a few tests though:
- what if a valid article is requested but it just doesn't have any comments yet?
- what if the article id is not the right format (a string instead for example)